mtgjson5/providers/scryfall/monolith.py

```diff
@@ -22,6 +27,83 @@
 LOGGER = logging.getLogger(__name__)
 
 
+BulkDataType = Literal[
+    "oracle_cards", "unique_artwork", "default_cards", "all_cards", "rulings"
+]
+
+
+class BulkDataObject(BaseModel):
+    """Represents a single bulk data file available from Scryfall"""
+
+    object: Literal["bulk_data"] = Field(description="Type identifier")
+    id: UUID = Field(description="Unique identifier for this bulk data")
+    type: BulkDataType = Field(description="Type of bulk data")
+    updated_at: datetime = Field(description="When this file was last updated")
+    uri: HttpUrl = Field(description="API URI for this bulk data object")
+    name: str = Field(description="Human-readable name")
+    description: str = Field(description="Description of what this file contains")
+    size: int = Field(description="Size in bytes of the compressed file")
+    download_uri: HttpUrl = Field(description="Direct download URL for the file")
+    content_type: str = Field(description="MIME type of the file")
+    content_encoding: str = Field(description="Compression encoding (usually gzip)")
+
+    def download(
+        self, session: Optional[requests.Session] = None, timeout: int = 300
+    ) -> List[Dict[str, Any]]:
+        """Download the bulk data file.
+
+        Uses a plain requests.Session to avoid caching large bulk files
+        that can exceed SQLite's blob size limits.
+        """
+        download_session = requests.Session()
```

Comment:
I see you're using the requests session instead of the retryable_seession.Session() we have defined in the code (which takes advantage of the local caching opportunity). Was this intentional?

Reply:
Thanks for the comment on this, makes sense!